src/meson.build

```diff
@@ -0,0 +1,75 @@
+src_directory = include_directories('.')
+
+vulkan = dependency('vulkan')
```

Comment:
It's just about style, but Meson recommends naming dependency variable `<project>_dep`, so here `vulkan_dep` would be better.